test/test_tdvp.jl

```diff
@@ -41,8 +41,6 @@ using Test: @test, @testset
   @test ITensors.scalartype(ψ1) == complex(elt)
   @test ψ1 ≈ tdvp(-time_step, H, ψ0; nsweeps=1, cutoff, nsite=1)
   @test ψ1 ≈ tdvp(H, ψ0, -time_step; nsweeps=1, cutoff, nsite=1)
-  #Different backend solvers, default solver_backend = "exponentiate"
-  @test ψ1 ≈ tdvp(H, ψ0, -time_step; nsweeps=1, cutoff, nsite=1, solver_backend="applyexp")
```

Comment:
I had this test here just as a regression test that it still runs if a user sets `solver_backend="applyexp"`, even though it calls `"exponentiate"` in the background.

Reply:
Good to know – I'lll put it back.